free-2.md

```diff
@@ -58,12 +105,26 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+With this project I feel that 90% of the project would be trying to engineer the
+***language*** aspects for it. This is because there already is a DSL called Make
+which I am just tyring to improve the language aspects of. The parts of the project
+that arent about language will mainly just be trying to parse the new fluency into the
+current Make file fluency. 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is probably a very ambitious project. Make is a very large, very well developed,
+and very old DSL so trying to properly port over all its features into a new lanugage/fluency
+would be extremely difficult and large. The project would probably have to focus on a certain
```

Comment:
Great work looking at a possible downside of this project and seeing ways around this downside. Like you've mentioned, people use make because that's the norm, but that doesn't make it the infallible. However, it can make it difficult to parse because it's so bloated. Looking at one part seems to be a great solution.